src/components/Accordion/Accordion.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactTestRenderer from 'react-test-renderer';
+import Accordion from './Accordion';
+import AccordionItem from './AccordionItem';
+
+describe('Accordion component should render expected markup', () => {
```

Comment:
I don't if you usually do it like this, but in my opinion `describe` shouldn't have `should` in the description.
It should look like this:
```js
describe('The Accordion component', () => {
  describe('<Function or life cycle you want to test (render here?)>', () => {
    it('should compare the component with a snapshot', () => {
      ...
    });
  });
});